kokoro/linux/java_jdk11/continuous.cfg

```diff
@@ -0,0 +1,11 @@
+# Config file for running tests in Kokoro
+
+# Location of the build script in repository
+build_file: "protobuf/kokoro/linux/java_linkage_monitor/build.sh"
```

Comment:
I think this should be: `protobuf/kokoro/linux/java_jdk11/build.sh`

Reply:
done